### DISCUSSION TITLE
install lsp plugins , as follows error

### DISCUSSION THREAD

**Discussion by wxzvim** [ayamir/nvimdots#861] [Q&A] [answered]
![image](https://github.com/ayamir/nvimdots/assets/110797126/84c1a0c6-4a4f-486b-aac4-1cc62c3aa0a8)

**Comment by wxzvim** [comment]
how to slove question? i want to reinstall ,but not find reinstall button

**Reply by Jint-lzxy** [reply, marked as answer]
You need to uninstall the package (if installed, press `X` while placing your cursor on `json-lsp`\[the text\]) before installing it (`i`). To force quit previous install procedure _(if the detached executable is no longer under your control)_:
```sh
sudo killall npm
```
